ctwrap/simulation.py

```diff
@@ -84,7 +100,15 @@ def run(self, name: Optional[str] ='defaults',
             config: Optional[Dict[str, Any]] = None,
             **kwargs: str) -> None:
         """
-        Run function holding configuration in dictionary.
+        Runs the simulation module's run method.
+
+        The :meth:`run` is used to call the module's run method.
+        A simple usage is
+
+        .. code-block:: Python
+
+           # calls the run method in module 'minimal`
```

Comment:
I think every time you mention module, replace it by 'simulation module' for clarity